maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gittest/src/main/java/org/apache/maven/scm/provider/git/command/checkin/GitCheckInCommandTckTest.java

```diff
@@ -102,6 +105,35 @@ public void testRejectedNonFastForwardPush() throws Exception {
                 "check-in should have been rejected since fast forward was not possible", checkInScmResult.isSuccess());
     }
 
+    @Test
+    public void testCommitNoVerify() throws Exception {
+        GitScmTestUtils.setupRejectAllCommitsPreCommitHook(getWorkingCopy());
+        GitScmTestUtils.setDefaultGitConfig(getWorkingCopy());
+        ScmFileSet addedFile = createWorkspaceChange(getWorkingCopy());
+        CheckInScmResult result;
```

Comment:
This should be separate variables inside the try blocks

Reply:
Done